eberron.md

```diff
@@ -95,13 +104,14 @@ When cast at 9th level or above, your range spans the entire plane.
 #### Bloodied Blade
 *2nd level*
 
-- **Casting Time:** 5 AP, bladed weapon
+- **Casting Time:** 3 AP, bladed weapon
 - **Range:** self
-- **Duration:** instant
+- **Duration:** 1 minute
+- **Damage Type:** slashing
 
 On casting you take a wound in unblockable damage. Your weapon becomes empowered, allowing you to use your spellcasting roll in place of your attack roll and giving it Reach 1. Your next attack hits every creature in your zone of control, dealing 1 wound and 2 wound dice in slashing damage.
 
-For each level cast above 2nd, add a wound and a wound die in slashing damage.
+For each level cast above 2nd, add a wound die in slashing damage.
```

Comment:
In general for AoE spells damage increases should only be a wound die. In some cases you can add a wound, but anything more than that makes it a very deadly spell that outpaces 8th-9th level spells. For example, Black Lightning is an 8th level spell and that one only deals 1 wound and 6 wound dice.